### ISSUE TITLE
Editor: <dmn:itemDefinition> with more than 10 <dmn:itemComponent> appears in other section on the UI

### ISSUE THREAD

**Issue by kauereinbold** [apache/incubator-kie-tools#1273]
**Describe the Bug**

After adding more than 10 items under a Data Type, the child elements appear expanded over other data types.

**Steps to Reproduce**

1. Open a DMN on the Editor VSCode/Kie sandbox;
2. Go to Data Types;
3. Add more than 10 items;
4. Go to the Editor;
5. Go back to Data Types;
6. You will see the items going over other types;

![image](https://user-images.githubusercontent.com/22650820/195410770-04ff9266-4547-4a08-979b-975e09dee758.png)

I created a repo with a sample with this problem [Traffic Violation - Many items.dmn](https://github.com/KaueReinbold/dmn-quarkus-example/blob/main/src/main/resources/Traffic%20Violation%20-%20Many%20items.dmn).

**Expected Behavior**

The Data Types should be properly organized to be accessed. 

**Environment**

- OS: Windows
- Visual Studio Code version: 1.71.2
- DMN Editor plugin: 0.23.0
- Language Support for Java(TM) by Red Hat v1.11.0

**Comment by yesamer** [comment]
https://issues.redhat.com/browse/KOGITO-8142

**Comment by yesamer** [comment]
Hi @KaueReinbold. 
I tried to reproduce your issue and:
- The DMN file you posted above is invalid, thus I can't open it.
- The issue doesn't reproduce in my local following your steps, please check the below video:

https://user-images.githubusercontent.com/16005046/197497854-1145237b-9f74-4296-bbf9-3ff04a60b35a.mov

Are some steps missing maybe?
Thank you.

**Comment by kauereinbold** [comment]
Hello @yesamer, 
Just to avoid versioning conflict with VSCode and OS. I used the [Kie Sandbox](https://sandbox.kie.org/) and I recorded this video:

https://user-images.githubusercontent.com/22650820/197552047-c8c021f8-6670-4274-b4cc-4c0ec1596fba.mp4

After adding more than 10 types in a structure, collapsing, and expanding you will see the Data Type structure overlaps others created in the UI only.

Here is the file created on this video:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<dmn:definitions xmlns:dmn="http://www.omg.org/spec/DMN/20180521/MODEL/" xmlns="https://kiegroup.org/dmn/_CC9A76B9-82DA-4907-B4F9-54B593A036EC" xmlns:feel="http://www.omg.org/spec/DMN/20180521/FEEL/" xmlns:kie="http://www.drools.org/kie/dmn/1.2" xmlns:dmndi="http://www.omg.org/spec/DMN/20180521/DMNDI/" xmlns:di="http://www.omg.org/spec/DMN/20180521/DI/" xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/" id="_F0B58EB7-B57B-47C8-9423-FF25486DBD2B" name="Untitled" typeLanguage="http://www.omg.org/spec/DMN/20180521/FEEL/" namespace="https://kiegroup.org/dmn/_CC9A76B9-82DA-4907-B4F9-54B593A036EC">
  <dmn:extensionElements/>
  <dmn:itemDefinition id="_7A5DC528-D94F-4D1C-A634-2C8B531C274F" name="Insert a name" isCollection="false">
    <dmn:itemComponent id="_CC5ED9FA-CD81-45F0-B2AB-55CC59F2CF41" name="Insert a name - 15" isCollection="false">
      <dmn:typeRef>Any</dmn:typeRef>
    </dmn:itemComponent>
    <dmn:itemComponent id="_E4AEE202-CF26-4305-A66D-07EFC41BCCB8" name="Insert a name - 14" isCollection="false">
      <dmn:typeRef>Any</dmn:typeRef>
    </dmn:itemComponent>
    <dmn:itemComponent id="_8AF3C125-FF88-402C-869B-9468BD565A68" name="Insert a name - 13" isCollection="false">
      <dmn:typeRef>Any</dmn:typeRef>
    </dmn:itemComponent>
    <dmn:itemComponent id="_6286700D-D39F-411B-9614-DCAFB6E00CBD" name="Insert a name - 12" isCollection="false">
      <dmn:typeRef>Any</dmn:typeRef>
    </dmn:itemComponent>
    <dmn:itemComponent id="_309FD120-821F-4907-83E0-217107DFF850" name="Insert a name - 11" isCollection="false">
      <dmn:typeRef>Any</dmn:typeRef>
    </dmn:itemComponent>
    <dmn:itemComponent id="_8B366DC7-88F1-4CFA-9EF2-79CBA9560ED0" name="Insert a name - 10" isCollection="false">
      <dmn:typeRef>Any</dmn:typeRef>
    </dmn:itemComponent>
    <dmn:itemComponent id="_0B034FCC-5D58-457A-B85E-F58F0789A607" name="Insert a name - 9" isCollection="false">
      <dmn:typeRef>Any</dmn:typeRef>
    </dmn:itemComponent>
    <dmn:itemComponent id="_BB8076EF-C800-4633-ABAC-7F281E58ED00" name="Insert a name - 8" isCollection="false">
      <dmn:typeRef>Any</dmn:typeRef>
    </dmn:itemComponent>
    <dmn:itemComponent id="_15F424B3-575D-47A3-9A76-226EE16D24BC" name="Insert a name - 7" isCollection="false">
      <dmn:typeRef>Any</dmn:typeRef>
    </dmn:itemComponent>
    <dmn:itemComponent id="_164641D2-7237-49FD-85AF-0ECFEB9D1DF6" name="Insert a name - 6" isCollection="false">
      <dmn:typeRef>Any</dmn:typeRef>
    </dmn:itemComponent>
    <dmn:itemComponent id="_6B572567-CA2F-4A9E-9CB1-3C1A6E19DF82" name="Insert a name - 5" isCollection="false">
      <dmn:typeRef>Any</dmn:typeRef>
    </dmn:itemComponent>
    <dmn:itemComponent id="_6668F1F0-F425-4635-9151-8DE197CF4C15" name="Insert a name - 4" isCollection="false">
      <dmn:typeRef>Any</dmn:typeRef>
    </dmn:itemComponent>
    <dmn:itemComponent id="_B4CFF711-2E07-4987-BB8C-494D9A43B9C8" name="Insert a name - 3" isCollection="false">
      <dmn:typeRef>Any</dmn:typeRef>
    </dmn:itemComponent>
    <dmn:itemComponent id="_02AC09ED-D105-4457-8802-C0DD2EDC11C7" name="Insert a name - 2" isCollection="false">
      <dmn:typeRef>Any</dmn:typeRef>
    </dmn:itemComponent>
    <dmn:itemComponent id="_26506BE9-58C1-48AD-8FB4-969B984EA531" name="Insert a name" isCollection="false">
      <dmn:typeRef>Any</dmn:typeRef>
    </dmn:itemComponent>
  </dmn:itemDefinition>
  <dmn:itemDefinition id="_CC193DE7-631B-47DB-8B24-129E384530E2" name="Insert a name - 2" isCollection="false">
    <dmn:itemComponent id="_CCA6797C-8FBB-49C1-97C8-4935F83B030D" name="Insert a name - 6" isCollection="false">
      <dmn:typeRef>Any</dmn:typeRef>
    </dmn:itemComponent>
    <dmn:itemComponent id="_A6F35BBE-9CC6-44BA-A138-B8432E5B3F9B" name="Insert a name - 5" isCollection="false">
      <dmn:typeRef>Any</dmn:typeRef>
    </dmn:itemComponent>
    <dmn:itemComponent id="_C4B23E48-5241-46E9-AB2A-2D00F241115B" name="Insert a name - 4" isCollection="false">
      <dmn:typeRef>Any</dmn:typeRef>
    </dmn:itemComponent>
    <dmn:itemComponent id="_3742AF2A-0C26-4409-B857-50C6F047B2F4" name="Insert a name - 3" isCollection="false">
      <dmn:typeRef>Any</dmn:typeRef>
    </dmn:itemComponent>
    <dmn:itemComponent id="_49A4BB08-E929-4745-BABF-D491CF027E57" name="Insert a name - 2" isCollection="false">
      <dmn:typeRef>Any</dmn:typeRef>
    </dmn:itemComponent>
    <dmn:itemComponent id="_7EEAF1D9-CB4A-4592-9067-8AD4CF59A76C" name="Insert a name" isCollection="false">
      <dmn:typeRef>Any</dmn:typeRef>
    </dmn:itemComponent>
  </dmn:itemDefinition>
  <dmndi:DMNDI>
    <dmndi:DMNDiagram id="_523D07EA-6732-472E-9B56-98EAF948A5D9" name="DRG">
      <di:extension>
        <kie:ComponentsWidthsExtension/>
      </di:extension>
    </dmndi:DMNDiagram>
  </dmndi:DMNDI>
</dmn:definitions>
```

Let me know if I can help with something else.

**Comment by yesamer** [comment]
@KaueReinbold Now I'm able to reproduce the issue, thank you. The missing part was the Structure item collapse.

**Comment by jomarko** [comment]
This is duplicated also by https://issues.redhat.com/browse/DROOLS-5636

**Comment by yesamer** [comment]
@KaueReinbold This issue has been fixed by @jomarko, it will be available from the next version, 0.27.0.
